### ISSUE TITLE
ESLint `that.getFilename` is not a function

### ISSUE THREAD

**Issue by zachary-russell** [WebDevStudios/WDS-Coding-Standards#68]
I am having an issue using ESLint. I'm getting an error anytime I try to run in both my editor and in the CLI. I tried searching this issue on ESLint itself and found no references that could help so I'm pretty sure it is related to this specific config. Any idea what's going wrong or how it can be fixed?

Note: I ran `npm install -g` in the coding standards eslint directory.

My Setup:

- OSX
- Node 10.15.3
- ESLint 5.16.0

**Example Command:**
`eslint --config=/Users/zacharyrussell/.composer/vendor/webdevstudios/wds-coding-standards/WebDevStudios/.eslintrc.js bar.js`

**Error Output:**
```
TypeError: that.getFilename is not a function
Occurred while linting /Users/zacharyrussell/bar.js:1
at Object.wdscsUtils.isThemeFile (/Users/zacharyrussell/.composer/vendor/webdevstudios/wds-coding-standards/WebDevStudios/eslint-plugin-webdevstudios/index.js:156:22)
    at * (/Users/zacharyrussell/.composer/vendor/webdevstudios/wds-coding-standards/WebDevStudios/eslint-plugin-webdevstudios/index.js:309:23)
    at listeners.(anonymous function).forEach.listener (/usr/local/lib/node_modules/eslint/lib/util/safe-emitter.js:45:58)
    at Array.forEach (<anonymous>)
    at Object.emit (/usr/local/lib/node_modules/eslint/lib/util/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/usr/local/lib/node_modules/eslint/lib/util/node-event-generator.js:251:26)
    at NodeEventGenerator.applySelectors (/usr/local/lib/node_modules/eslint/lib/util/node-event-generator.js:278:22)
    at NodeEventGenerator.enterNode (/usr/local/lib/node_modules/eslint/lib/util/node-event-generator.js:294:14)
    at CodePathAnalyzer.enterNode (/usr/local/lib/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:632:23)
    at nodeQueue.forEach.traversalInfo (/usr/local/lib/node_modules/eslint/lib/linter.js:752:32)
```

**Comment by aubreypwd** [comment]
A fix for this is on the `release-2.0.1` branch and will be going out as a hotfix soon!

**Comment by aubreypwd** [comment]
Note, pending https://github.com/WebDevStudios/WDS-Coding-Standards/pull/76
